src/System.Text.Primitives/System/Text/Extensions.Integer.cs

```diff
@@ -1,7 +1,9 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace System.Text
+using System.Buffers;
+
+namespace System.Buffers.Text
```

Comment:
Why are the TryFormat extension methods in the System.Buffers.Text namespace when the Custom Formatter TryFormat are in System.Buffers namespace? Should the Extensions class also be in System.Buffers namespace?